psi4/src/psi4/cctransort/memcheck.cc

```diff
@@ -44,7 +44,7 @@ void memcheck(int reference) {
         for (int h = 0; h < Z.params->nirreps; h++) {
             irrep_size = (size_t)Z.params->rowtot[h] * Z.params->coltot[h];
             size += irrep_size;
-            outfile->Printf("\tSize of irrep %1lu of <ab|cd> integrals: %10.3lf (MW) / %10.3lf (MB)\n", h,
+            outfile->Printf("\tSize of irrep %d of <ab|cd> integrals: %10.3lf (MW) / %10.3lf (MB)\n", h,
```

Comment:
I had a habit of using `%zu` for `size_t`, and https://stackoverflow.com/a/2524675 concurs. Is there an advantage to `%d` here? I see you're using some `%zu`s below.

Reply:
As far as I could gather, `%zu` is not an "universal automatic-size integer format", but "the format that works for size_t integers". And in this case, the type of `h` is just `int`, so unless `size_t` happens to be `int`, `%zu` would not be appropriate here, I think.

Reply:
Ack, I was looking at `irrep_size`, not `h`, for the var-to-be-formatted. Sorry, you're right.